### ISSUE TITLE
Unable to resolve path aliases

### ISSUE THREAD

**Issue by GHNewbiee** [sarsamurmu/reboost#42]
*reboost: 0.16.1*

**reboost.js**
```js
const { start } = require('reboost');

start({
  entries: [
    ['./src/components/my-element/index.js', './public/dist/index.js']
  ],
  contentServer: {
    root: './public',
    open: true
  },
  resolve: {
    alias: {
      Icons: './public/dist/assets/icons'
    }
  }
});
```
**index.js**
```js
import { a, b, c } from 'Icons/icons.js';
...
```
Unfortunately, it fails displaying `Unable to resolve path "Icons/icons" of ".../index.js`

But, by using
```js
import { a, b, c } from '../../../public/dist/assets/icons/icons.js';
```
it works as expected.

Do I miss something? Are there any prerequisites?

**Comment by sarsamurmu** [comment]
It's a bug in Reboost. Aliases are just getting substituted, without fixing relative paths. So basically it's now turning this
```js
import 'Icons/icons.js';
```
into
```js
import './public/dist/assets/icons/icons.js';
```

For temporary fix you can pass absolute paths in the option, like this
```js
const path = require('path');
const { start } = require('reboost');

start({
  resolve: {
    alias: {
      Icons: path.join(__dirname, './public/dist/assets/icons')
    }
  }
});
```

This should be fixed in the next release.

**Comment by sarsamurmu** [comment]
Fixed in `v0.16.2`
